javascript/javascript2/week1/homework app2/main.js

```diff
@@ -0,0 +1,27 @@
+console.log("Script loaded");
+
+const products = getAvailableProducts();
+console.log(products);
+
+let ul = document.querySelector(".listOfProducts");
+
+function renderProducts(ourProducts) {
+    for (let i = 0; i < ourProducts.length; i++) {
+        const productNames = ourProducts[i];
+        const productTitle = document.createElement("h3");
+        const productPrice = document.createElement("p");
+        const productRating = document.createElement("p");
+        productTitle.innerText = productNames.name;
+        productPrice.innerText = "Price :" + productNames.price;
+        productRating.innerText = "Rating :";
```

Comment:
in here you have an extra ";" probably a typo as you are missing a "+", that prevents the ratings from showing